sql/moz-fx-data-shared-prod/firefox_ios/new_profile_activation/view.sql

```diff
@@ -1,7 +1,7 @@
 CREATE OR REPLACE VIEW
-  `moz-fx-data-shared-prod.firefox_ios.new_profile_activation`
+  firefox_ios.new_profile_activation
 AS
 SELECT
   *
 FROM
-  `moz-fx-data-shared-prod.firefox_ios_derived.new_profile_activation_v1`
+  firefox_ios_derived.new_profile_activation_v1
```

Comment:
Once the table is generated and the backfill catches up this will get changed to v2. Could be done during the v1 deprecation process.